### ISSUE TITLE
Margin orders

### ISSUE THREAD

**Issue by evo11x** [sonvister/Binance#126]
Hi,
   Is there a way to place margin orders with this api?
Thanks!

**Comment by sonvister** [comment]
No, and I don't plan on adding that capability in the foreseeable future. Pull requests are welcome.
